demoapp/src/main/java/com/transferwise/tasks/demoapp/Application.java

```diff
@@ -4,11 +4,9 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.ApplicationPidFileWriter;
-import org.springframework.transaction.annotation.EnableTransactionManagement;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 
 @SpringBootApplication
-@EnableTransactionManagement
```

Comment:
This is handled by gaffer starter now right?

Reply:
Yep,
```
@EnableTransactionManagement
public class GafferJtaConfiguration {
```